BitwardenShared/Core/Platform/Models/Domain/EnvironmentUrls.swift

```diff
@@ -43,11 +43,11 @@ extension EnvironmentUrls {
     ///
     init(environmentUrlData: EnvironmentUrlData) {
         if let base = environmentUrlData.base {
-            apiURL = base.appendingPathComponent("/api")
+            apiURL = base.appendingPathComponent("api")
```

Comment:
`appendingPathComponent(_:)` adds the slash if it doesn't exist so we don't need to manually add one here.